.github/CONTRIBUTING.md

```diff
@@ -25,9 +25,12 @@ Facebook has a [bounty program](https://www.facebook.com/whitehat/) for the safe
 disclosure of security bugs. In those cases, please go through the process
 outlined on that page and do not file a public issue.
 
-## Coding Style  
-* 4 spaces for indentation rather than tabs
-* 80 character line length
+## Coding Guidelines and Development 
+* Ensure you stick to the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/)
+* Whenever possible please fix pre-existing code standards errors in the files that you change. It is ok to skip that for larger files or complex fixes.
+* Ensure that your code supports the minimum supported versions of PHP and WordPress; this is shown at the top of the `readme.txt` file.
+* Make sure to write good and detailed commit messages (see [this post](https://chris.beams.io/posts/git-commit/) for more on this) and follow all the applicable sections of the pull request template.
+* Please avoid modifying the changelog directly or updating the .pot files.
```

Comment:
Let's confirm the changelog process - cc @danielbitzer who is drafting a release process right now. I'm not certain .. in ventures the current process may be to edit the changelog in the PR. 

For the record, I think doing the changelog in the PR description (release DRI collates and edits changelog) is a good approach.

Reply:
> I think doing the changelog in the PR description (release DRI collates and edits changelog) is a good approach.

I like either approach, PR description or modifing the changelog directly. I think I slightly prefer the modify changelog approach for efficiency. I do think we need to get better at writing changelog entries though.